templates/components/vectordbs/python/astra/generate.py

```diff
@@ -32,6 +32,6 @@ def generate_datasource():
     logger.info(f"Successfully created embeddings in the AstraDB")
 
 
-if __name__ == "__main__":
+def main():
```

Comment:
I think we should keep the `if __name__ == "__main"` to allow running generate without poetry script.

Reply:
good idea, @leehuwuj what's best practice here?

Reply:
I think we can change the generate file like this
```python
def generate_datasource():
    init_settings()
    # do the rest of generation
    ...

if __name__ == "__main__":
    generate_datasource()
```

and update the scripts in `pyproject.toml` to this
```
[tool.poetry.scripts]
generate = "app.engine.generate:generate_datasource"
```